config/owasp/suppressions.xml

```diff
@@ -2,6 +2,49 @@
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
     <suppress>
         <notes><![CDATA[
+   file name: checkstyle-8.10.1.jar
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/com\.puppycrawl\.tools/checkstyle@.*$</packageUrl>
+        <vulnerabilityName>CVE-2019-10782</vulnerabilityName>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: org.apache.sling.javax.activation-0.1.0.jar
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.apache\.sling/org\.apache\.sling\.javax\.activation@.*$</packageUrl>
+        <cve>CVE-2016-5394</cve>
+        <cve>CVE-2016-6798</cve>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: selenium-opera-driver-3.141.59.jar
+   ]]></notes>
+        <sha1>6b11ed01182bfd8bb62ddeae34cf9eadd3cfc7ff</sha1>
+        <cve>CVE-2003-1561</cve>
+        <cve>CVE-2008-1761</cve>
+        <cve>CVE-2008-1764</cve>
+        <cve>CVE-2008-3079</cve>
+        <cve>CVE-2008-3172</cve>
+        <cve>CVE-2008-4293</cve>
+        <cve>CVE-2008-4695</cve>
+        <cve>CVE-2008-4696</cve>
+        <cve>CVE-2008-4794</cve>
+        <cve>CVE-2008-4795</cve>
+        <cve>CVE-2008-5679</cve>
+        <cve>CVE-2009-0915</cve>
+        <cve>CVE-2010-5227</cve>
+        <cve>CVE-2015-8960</cve>
+        <cve>CVE-2019-19788</cve>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
+   This suppresses a CVE-2020-5529 cve in any htmlunit of selenium
+   ]]></notes>
+        <cpe>cpe:/a:htmlunit:htmlunit</cpe>
+        <cve>CVE-2020-5529</cve>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
```

Comment:
These Suppressions are already in Master, moving them here to resolve dependency-check failure in Jenkins